homeassistant/components/livisi/coordinator.py

```diff
@@ -69,14 +70,15 @@ async def async_setup(self) -> None:
                 livisi_connection_data=livisi_connection_data
             )
         controller_data = await self.aiolivisi.async_get_controller()
-        if controller_data["controllerType"] == "Avatar":
+        if controller_data["controllerType"] == AVATAR:
             self.port = AVATAR_PORT
             self.is_avatar = True
         else:
             self.port = CLASSIC_PORT
             self.is_avatar = False
+        controller_type = controller_data["controllerType"]
         self.serial_number = controller_data["serialNumber"]
-        self.controller_type = controller_data["controllerType"]
+        self.controller_type = controller_type
```

Comment:
```suggestion
```

Reply:
Did you mean to remove `self.serial_number = controller_data["serialNumber"]` ?

Reply:
I don't understand this comment. Should I remove the serial number assignment?